src/ansys/mapdl/core/examples/downloads.py

```diff
@@ -103,23 +94,28 @@ def _retrieve_file(url, filename, _test=False):
     local_path = os.path.join(pymapdl.EXAMPLES_PATH, os.path.basename(filename))
     local_path_no_zip = local_path.replace(".zip", "")
     if os.path.isfile(local_path_no_zip) or os.path.isdir(local_path_no_zip):
-        return local_path_no_zip, None
+        return local_path_no_zip
 
     # Perform download
-    saved_file, resp = urllib.request.urlretrieve(url)
-    shutil.move(saved_file, local_path)
+    requested_file = requests.get(url, timeout=10)
+    requested_file.raise_for_status()
+    file_content = requested_file.content
+
+    with open(local_path, "wb") as f:
+        f.write(file_content)
```

Comment:
Why store in variable for later writing to file only? Do it directly
```suggestion

    with open(local_path, "wb") as f:
        f.write(requested_file.content)
```